easybuild/easyconfigs/k/KofamScan_data/KofamScan_data-20230401-gompi-2021b.eb

```
@@ -0,0 +1,50 @@
+easyblock = 'Binary'
+
+name = 'KofamScan_data'
+version = '20230401'
+
+homepage = 'https://www.genome.jp/tools/kofamkoala/'
+description = """KofamKOALA assigns K numbers to the user's sequence data by 
+HMMER/HMMSEARCH against KOfam (a customized HMM database of KEGG Orthologs 
+(KOs)).  K number assignments with scores above the predefined thresholds 
+for individual KOs are more reliable than other proposed assignments.  Such 
+high score assignments are highlighted with asterisks '*' in the output.  
+The K number assignments facilitate the interpretation of the annotation 
+results by linking the user's sequence data to the KEGG pathways and EC 
+numbers.  [data only]."""
+# software_license = 'MIT'
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = [
+    'https://www.genome.jp/ftp/db/kofam/',  # add archives/YYYY-MM-DD/ if moved
+]
+sources = [
+    'ko_list.gz',
+    'profiles.tar.gz',
+]
```

This will be problematic; the download source file doesn't contain any information on what version this is.
In fact, I can't tell if this is a 20230401 version from anything here, in fact, the time stamps on the ftp states 24-Apr-2023 and 26-Apr-2023, so it's not clear what this is supposed to be.

We'll have to do a few things, use `filename` and `download_filename` to indicate that these differ.
Something along the lines of
```suggestion
source_urls = [
    'https://www.genome.jp/ftp/db/kofam/%(version)s',  # prefer the archive url present after new releases
    'https://www.genome.jp/ftp/db/kofam/',
]
sources = [
    {'download_filename': 'ko_list.gz', 'filename': 'ko_list_%(version)s.gz'}
    {'download_filename': 'profiles.tar.gz', 'filename': 'profiles_%(version)s.tar.gz'}
]
```
might be sufficient to protect reproducibility here, but I had to guess what the version string will be in the future. 

Maybe this means we would prefer the 2023-04-26 version as the version string. 

(there would of course be several modifications that follow from this renaming)

---

Thank you. I did know that we can do this filename versioning locally. Will take the codes in the next commit. I will also take "2023-04-26" as the version string, but not sure how will KofamScan name the current database version when they move files into the archive.